src/Control/Comonad.hs

```diff
@@ -3,6 +3,9 @@
 {-# LANGUAGE DeriveDataTypeable, StandaloneDeriving, Safe, DefaultSignatures #-}
 #elif __GLASGOW_HASKELL__ >= 702
 {-# LANGUAGE Trustworthy, DefaultSignatures #-}
+#endif
+#if __GLASGOW_HASKELL__ >= 706
+{-# LANGUAGE KindSignatures, PolyKinds #-}
```

Comment:
No need to enable `KindSignatures`—that's implied by `PolyKinds`.

Reply:
Thanks! Fixed.